lib/css_injector.js

```diff
@@ -17,7 +17,7 @@ CSSInjector.prototype = {
     this.filePathsToInject.forEach((p) => {
       if (inputHTML.indexOf(p) === -1) { return; }
 
-      let regex = new RegExp(`<link rel="stylesheet" href="[^"]*${p}"[^>]*>`)
+      let regex = new RegExp(`<link (\\w+)=("|'| |)(.*)("|'| ) href="[^"]*${p}"[^>]*>`)
```

Comment:
hm I think `<link[^>]* href="[^"]*${p}"[^>]*>` would be fine right? And would permit any number of attributes.

Reply:
I'll try that.